chain/network/src/routing/routing_table_actor.rs

```diff
@@ -408,10 +429,38 @@ impl RoutingTableActor {
 impl Handler<StopMsg> for RoutingTableActor {
     type Result = ();
     fn handle(&mut self, _: StopMsg, _ctx: &mut Self::Context) -> Self::Result {
+        self.edge_verifier_pool.do_send(StopMsg {});
         System::current().stop();
     }
 }
 
+impl Handler<EdgeList> for RoutingTableActor {
```

Comment:
add comment on what this handler does

Reply:
See updated comments in https://github.com/near/nearcore/pull/5429